pkg/instance/instance.go

```diff
@@ -282,25 +277,10 @@ func (i *Instance) SetGitRepo(ctx context.Context, gitContext builder.GitContext
 	return i.builderFactory.BuildImageFromGitRepo(ctx, gitContext, imageName)
 }
 
-// SetImageInstant sets the image of the instance without a grace period.
-// Instant means that the pod is replaced without a grace period of 1 second.
-// It is only allowed in the 'Running' state.
-func (i *Instance) SetImageInstant(ctx context.Context, image string) error {
-	if !i.IsState(StateStarted) {
-		return ErrSettingImageNotAllowedForSidecarsStarted.WithParams(i.state.String())
-	}
-
-	if i.isSidecar {
-		return ErrSettingImageNotAllowedForSidecars
-	}
-
-	return i.setImageWithGracePeriod(ctx, image, nil)
-}
-
 // SetCommand sets the command to run in the instance
 // This function can only be called when the instance is in state 'Preparing' or 'Committed'
 func (i *Instance) SetCommand(command ...string) error {
-	if !i.IsInState(StatePreparing, StateCommitted) {
+	if i.IsInState(StateStarted, StateCommitted) {
```

Comment:
Once the Instance is started, `SetCommand` will not have any effect.
This function is used to set the command of which the container is started, which we need to be able to set in `StatePreparing`.